packages/ERTP/src/types.js

```diff
@@ -37,13 +37,13 @@
  */
 
 /**
- * @typedef {'nat' | 'set' } AmountMathKind
+ * @typedef {'nat' | 'set' } AssetKind
```

Comment:
I was going to rip off the bandaid and make this a breaking change completely, since we're reaching the point where deprecated things shouldn't be in ERTP and Zoe anymore. People will get lint errors if they use MathKind, so in that sense, it will be marked as deprecated.